crates/cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -4,6 +4,7 @@
 // SPDX-License-Identifier: MIT OR Apache-2.0
 
 use crate::parser::{
+    inherit::*,
```

Comment:
why not just include `ParsedInheritedMethod` ?

Reply:
Done